packages/server/lib/cloud/api/index.ts

```diff
@@ -508,6 +512,8 @@ export default {
   },
 
   updateInstanceArtifacts (options: UpdateInstanceArtifactsOptions, body: UpdateInstanceArtifactsPayload) {
+    debug('PUT %s %O', recordRoutes.instanceArtifacts(options.instanceId), body)
```

Comment:
I think we inadvertently put capital O in a couple of places in the protocol code, but I think the pattern I've seen elsewhere is the little o. It makes objects easier to copy and paste later.

```suggestion
    debug('PUT %s %o', recordRoutes.instanceArtifacts(options.instanceId), body)
```